styleguide/_themes/derek/scss/components/ticker.scss

```diff
@@ -17,11 +17,11 @@ $ticker-span-bg: rgba(200, 0, 0, .5);
 }
 
 .ticker-container {
-  @include make-container;
+  @include make-container-fluid;
```

Comment:
As illustrated here, the general idea across the board is that all containers should be swapped to container-fluid. With this being such a sweeping change, I imagine some heavy QE will be necessary for this to make sure nothing else breaks in the process.